lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -9,7 +9,12 @@ def fn_name(): return sys._getframe(1).f_code.co_name
 try:
     import gi
 except ImportError:
-    raise ImportError("Gtk3 backend requires pygobject to be installed.")
+    try:
+        import pgi
+        pgi.install_as_gi()
```

Comment:
Well the code exists on github, https://github.com/lazka/pgi/blob/master/pgi/__init__.py

it basically just does:
```python
import pgi
sys.modules["gi"] = pgi
```